supervisor/supervisor.go

```diff
@@ -125,15 +125,23 @@ func (s *Supervisor) ScheduleAdhoc(a *db.Task) {
 		// expect a JobUUID to move to the schedq Immediately
 		for _, job := range s.jobq {
 			if !uuid.Equal(job.UUID, a.JobUUID) {
+
```

Comment:
Either that or a side effect of me moving code around. Maybe just to bother a well-intending code reviewer. The world may never know.